### PR TITLE
Add an example demonstrating running WExT on Pan-Cancer in different modes

### PR DESCRIPTION
**Pull request by mdml**
See the changes in the `examples` directory. I won't provide further info on how to test, since the idea is that a naive user can figure it out and use the example for guidance.

### CODE REVIEW

File: examples/pancan/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+### Data ###
+To demonstrate these two modes, we generate simulated data. The data includes two implanted gene sets mutated in approximately 25% of samples.
+* `(G1, G2)`: this set has "across" mutual exclusivity, i.e. exclusivity where both genes are mutated in both datasets/cancer types.
+* `(G3, G4)`: this set has "between" mutual exclusivity, i.e. each gene is mutated in only one dataset/cancer type.
+
+The datasets also include other genes that are mutated in each sample at a fixed background mutation rate. We ensure that every sample has at least one mutation.
+
+### Usage ###
+
+Run `make` to process the mutations, compute mutation probabilities, and run the WRE test in both modes described above. You can find the two output TSV files in `output/`. Running on a single core, the example takes approximately 23 seconds.
```
Comment (matthewreyna) [inline examples/pancan/README.md]: It takes 21 seconds for me with `make K=2 NP=100` and about 5 minutes for me with `make K=3 NP=100`.  Let's say something like the above example should take under a minute running on a single core of a modern laptop or desktop.

File: examples/pancan/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+You can pass in a few optional parameters to `make`, including:
+* `K`: gene set size (default: 2)
+* `NP`: number of permutations (default: 100)
+* `NUM_GENES`: number of genes in simulated dataset (default: 50)
+* `NUM_SAMPLES`: number of samples in simulated dataset (default: 100)
+* `RANDOM_SEED`: random seed for PRNG (default: 2)
+* `BMR`: background mutation rate (default: 0.01)
+
+For example, to run the example searching for sets of size three with 100 permutations:
```
Comment (matthewreyna) [inline examples/pancan/README.md:31]: Would `make K=2 NP=100` be a better example?

File: examples/pancan/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+    make K=3 NP=100
+
+The easiest way to interpret the results is to compare the "between" set `(G3, G4)` P-value in the two output files in `output/`. The P-value for `(G3, G4)` should be lower in the "across" output file.
```
Comment (matthewreyna) [inline examples/pancan/README.md]: Can you clarify?  Did you mean to say that the P-value for `(G3, G4)` should be smaller in the "between" output file than the "across" output file?

**Comment by matthewreyna** [comment]
I ran the example, and the p-values for the implanted gene sets with "across" and "between" mutual exclusivity reflect the differences between the two tests.  I like the document-by-example approach, and the MAKE file is easy to change for actual experiments.  It would help to update the end of the documentation to more explicitly draw conclusions from the p-values for different gene sets and tests.

**Comment by mdml** [comment]
@matthewreyna: good comments. See updates in c548119.

**Comment by matthewreyna** [comment]
LGTM!

**Pull request merged**